lib/team_api/joiner.rb

```diff
@@ -138,7 +138,7 @@ def join_project_data
     def store_project_errors(project, errors)
       project['errors'] = errors
       name = project['github'][0] || project['name']
-      data['errors'][name] |= errors unless data['errors'].nil?
+      data['errors'][name] = (data['errors'][name] || []) | errors
```

Comment:
Can we just assign `data['errors'][name] = errors`? Or maybe we need `(data['errors'] ||= {})[name] = errors`?